### ISSUE TITLE
Lease Area None Type error - Environmental module

### ISSUE THREAD

**Issue by davidbould** [DTOcean/dtocean#5]
Users reporting an error from the Enviro module when running the floating wave example from the release database. Module appears to be having difficulty with the lease area.

Opened example data locally. Error reproduced. Lease area seems correct in plots.

Lease area variable appears to be getting wiped at some point during running in GUI. On loading sample floating wave from the database, the lease area variable is visible under both the Project Boundaries Interface and the Enviro Assessment, and values look sensible. After running the hydro module, this variable now appears to be blank and so the Enviro Assessment fails.

**Comment by H0R5E** [comment]
OK, I can confirm this one as well:

```
2017-03-06 19:33:02,369 - CRITICAL - dtocean_gui.main - An AttributeError occurred: 'NoneType' object has no attribute 'area'
2017-03-06 19:33:02,721 - CRITICAL - dtocean_gui.main -   File "C:\DTOcean\lib\site-packages\dtocean_gui\main.py", line 185, in run
    self._project)
  File "C:\DTOcean\lib\site-packages\dtocean_core\menu.py", line 480, in execute_current
    allow_non_execution=True)
  File "C:\DTOcean\lib\site-packages\dtocean_core\core.py", line 1834, in auto_execute
    register_level)
  File "C:\DTOcean\lib\site-packages\dtocean_core\core.py", line 1769, in execute_interface
    interface = core.connect_interface(project, interface)
  File "C:\DTOcean\lib\site-packages\dtocean_gui\core.py", line 267, in connect_interface
    interface = super(GUICore, self).connect_interface(project, interface)
  File "C:\DTOcean\lib\site-packages\dtocean_core\core.py", line 1244, in connect_interface
    interface.connect()
  File "C:\DTOcean\lib\site-packages\dtocean_core\interfaces\environmental.py", line 990, in connect
    lease_area = self.data.lease_boundary.area
```

**Comment by H0R5E** [comment]
So the issue is with the module scope evaluation which is masking the database outputs. This is unwanted behaviour and the results of the database interfaces should probably not be tagged with "output".

**Comment by H0R5E** [comment]
Fixed in PR DTOcean/dtocean-core#14
